.github/workflows/labels.yml

```
@@ -0,0 +1,19 @@
+# Automatically labels PRs based on the configuration file
+# you are probably looking for 👉 `.github/labeler.yml`
+name: Label PRs
+
+on:
+  - pull_request_target
+
+jobs:
+  triage:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: depot-ubuntu-24.04-arm-small
```

Not sure about this runner type, we don't have an account with https://depot.dev/docs/github-actions/runner-types

```suggestion
    runs-on: ubuntu-latest
```